src/metrikker/iatjenester.ts

```diff
@@ -10,24 +10,42 @@ interface IaTjenesteMetrikk {
     kilde: String;
     tjenesteMottakkelsesdato: String;
 }
+export enum IaTjenesteKilde {
+    SYKEFRAVÆRSSTATISTIKK = 'SYKEFRAVÆRSSTATISTIKK',
+    KALKULATOR = 'KALKULATOR',
+}
+const erMetrikkerSendtForKilde = (
+    orgnr: string | undefined,
+    sendteMetrikker: [TjenestePerOrgnr],
+    kilde: IaTjenesteKilde
+) => {
+    return sendteMetrikker.some(
+        (tjenestePerOrgnr) => tjenestePerOrgnr.orgnr === orgnr && tjenestePerOrgnr.kilde === kilde
+    );
+};
 
 export const erIaTjenesterMetrikkerSendtForBedrift = (
     orgnr: string | undefined,
-    sendteMetrikker: [string]
+    sendteMetrikker: [TjenestePerOrgnr],
+    kilde: IaTjenesteKilde = IaTjenesteKilde.SYKEFRAVÆRSSTATISTIKK
 ): boolean => {
     if (orgnr === undefined) {
         return true;
     } else {
-        return sendteMetrikker.includes(orgnr);
+        return erMetrikkerSendtForKilde(orgnr, sendteMetrikker, kilde);
```

Comment:
Er det ikke litt rart å svare "ja" på spørsmålet "er ia-tjenester sendt for bedrift" når vi ikke vet hvilken bedrift det er? 😅 

Uansett så kan logikken
```
  if (orgnr === undefined) {
      return true;
  } else {
      return erMetrikkerSendtForKilde(orgnr, sendteMetrikker, kilde);
  }
```
forenkles til
```
  return orgnr === undefined || erMetrikkerSendtForKilde(orgnr, sendteMetrikker, kilde);
```

kan forenkles til 
```
    return orgnr === undefined) {
        return true;
    } else {
        return erMetrikkerSendtForKilde(orgnr, sendteMetrikker, kilde);
```